lib/src/chart/scatter_chart/scatter_chart_painter.dart

```diff
@@ -290,6 +290,67 @@ class ScatterChartPainter extends AxisChartPainter<ScatterChartData> {
       );
     }
 
+    if (data.scatterLabelSettings.showLabel) {
+      for (final scatterSpot in data.scatterSpots) {
+        int spotIndex = data.scatterSpots.indexOf(scatterSpot);
+
+        String label = data.scatterLabelSettings.getLabelFunction(spotIndex);
+
+        if (label.isEmpty || !scatterSpot.show) {
+          continue;
+        }
+
+        final span = TextSpan(
+          text: label,
+          style: data.scatterLabelSettings.getLabelTextStyleFunction(spotIndex),
```

Comment:
We have a `Utils().getThemeAwareTextStyle()` function which helps us to support the app theme by default.
Please use it (check [here](https://github.com/imaNNeoFighT/fl_chart/blob/master/lib/src/chart/line_chart/line_chart_painter.dart#L917) for more info)

Reply:
Yes, just like other places that we passed context to our methods.